src/presentation/dtos/media-dto.ts

```diff
@@ -0,0 +1,12 @@
+import { Media } from '../../model/entities/media'
+
+export class MediaDto {
+
+  public readonly id: string
+  public readonly duration?: number
```

Comment:
Why is duration optional?

Reply:
We might not have finished loading the media on the media server and hence not have a duration yet. Atleast that is the idea.
But if Media is also for graphics, then it's not all graphics that have a duration